operators/volume_loader/nrrd_loader.cpp

```diff
@@ -234,4 +274,40 @@ bool load_nrrd(const std::string& file_name, Volume& volume) {
   return true;
 }
 
+std::vector<double> parse_vector(std::string str) {
+  std::vector<double> result;
+  trim(str);
+  // ensures the string is surround by parenthesis and then remove them
+  if ((str[0] != '(') || (str[str.length() - 1] != ')')) return result;
```

Comment:
Should this throw an error if there are no parenthesis?

Reply:
Thank you!